WISE-Community/WISE-Client
# Convert Graph student component to Angular

Make sure Graph still works for the student.

Particular things to test as a student
- Adding a point
- Dragging a point
- Deleting a point
- Make sure line plot works
- Make sure scatter plot works
- Make sure column plot works (make sure to author with Plot Type=Column Plot, Axis Type=Categories, and also set non zero values in the series pre populated data points)
- Having multiple series
- Having multiple trials
- Creating a New Trial
- Showing/hiding trials
- Deleting a trial
- Changing to another trial
- Changing to another series (make sure to author the Graph with multiple series)
- Resetting a series
- Undoing a change
- Adding to Notebook
- Changing x and y axis limits
- Multiple Y Axes
- Import from Graph to Graph
- Import from Graph to Table
- Import from Table to Graph
- Import from Embedded to Graph (like in https://wise.berkeley.edu/preview/unit/32121/node81)
- Import from Graph to Animation (like in https://wise.berkeley.edu/preview/unit/32657/node22 draw points, click save, then go to next step and play the Animation)
- Importing from Graph to Draw does not currently work which is a problem with Draw. I will fix this in another issue.

Closes #264
